lbrycrd/stuff/start.sh

```diff
@@ -6,35 +6,39 @@
 # ## not specified it will only create an index for transactions that are related to the wallet or have unspent outputs.
 # ## This is specific to chainquery.
 
+# The config file does not exist in the container image. It must be mounted, or
+# if not, a default config is generated using environment variables.
+CONFIG_PATH=/etc/lbry/lbrycrd.conf
+if [ -f "$CONFIG_PATH" ]
```

Comment:
CONFIG_PATH=${CONFIG_PATH:-/etc/lbry/lbrycrd.conf} would let you set a default value for the config path but allow an override if there was an environment variable set.

Otherwise you're likely just going to overwrite the variable when the bash hits line 11

Reply:
This is the path inside the container, not outside. So I don't think it needs to be configurable, does it?